src/lib-components/BlockEvent.vue

```diff
@@ -84,111 +84,106 @@ export default {
 
 <style lang="scss" scoped>
 .block-event {
-  position: relative;
-  z-index: 0;
-  background-color: var(--color-primary-blue-03);
-  color: var(--color-white);
-  font-family: var(--font-primary);
-  max-width: 800px;
-
-  .row {
-    position: absolute;
-    top: 0;
-    left: 0;
-    right: 0;
-
-    display: flex;
-    flex-direction: column;
-    flex-wrap: nowrap;
-    justify-content: center;
-    align-content: flex-start;
-    align-items: flex-start;
-  }
-
-  .molecule {
-    background-image: url("ucla-library-design-tokens/assets/svgs/molecule-placeholder.svg");
-    background-size: auto 100%;
-    height: 40%;
-  }
-
-  .category {
-    @include overline;
-    color: var(--color-secondary-yellow-01);
-    margin-bottom: var(--space-s);
-  }
-
-  .title {
-    @include step-2;
-    margin-bottom: var(--space-s);
-  }
-
-  .date-time {
-    display: flex;
-    flex-direction: column;
-    margin-bottom: var(--space-m);
-  }
-
-  .dates,
-  .time {
-    @include step-0;
-  }
-
-  .text {
-    bottom: 0;
-    top: auto;
-    max-width: 500px;
-    padding: 20px var(--unit-gutter);
-    margin: 0 auto;
-    height: 60%;
-  }
-
-  .button {
-    border: 1px solid var(--color-primary-blue-02);
+    position: relative;
+    z-index: 0;
     background-color: var(--color-primary-blue-03);
-    padding: 16px 40px;
-    @include button;
-    color: var(--color-theme);
-    transition: color 400ms ease-in-out;
-
-    // Themes
-    --color-theme: var(--color-primary-blue-02);
-
-    &.color-visit {
-      --color-theme: var(--color-visit-fushia-03);
+    color: var(--color-white);
+    font-family: var(--font-primary);
+    max-width: 800px;
+
+    .row {
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+
+        display: flex;
+        flex-direction: column;
+        flex-wrap: nowrap;
+        justify-content: center;
+        align-content: flex-start;
+        align-items: flex-start;
     }
 
-    &.color-help {
-      --color-theme: var(--color-help-green-03);
+    .molecule {
+        background-image: url("node_modules/ucla-library-design-tokens/assets/svgs/molecule-placeholder.svg");
```

Comment:
update this line to background-image: url("ucla-library-design-tokens/assets/svgs/molecule-placeholder.svg");